synfig-studio/src/gui/docks/dock_children.cpp

```diff
@@ -55,6 +57,9 @@ using namespace studio;
 Dock_Children::Dock_Children():
 	Dock_CanvasSpecific("children",_("Library"),Gtk::StockID("synfig-children"))
 {
+	// Make sure the used style is consistent with other docks
+	get_style_context()->add_class("synfigstudio-efficient-workspace");
+
```

Comment:
Since we already added that class to our parent widget ie., notebook, you don't need to set it here again

Only the CSS code is needed for this PR :)

![Screenshot_2021-06-04_08-49-27](https://user-images.githubusercontent.com/24214509/120741507-cc251e80-c512-11eb-9d30-9a929be55047.png)

Reply:
Instead of styling the overall dock, style only the toolbar container/widget: https://github.com/synfig/synfig/pull/2170#issuecomment-854363273
Here's an example of how I did it https://github.com/synfig/synfig/pull/2170#issuecomment-854375880